Cargo.toml

```diff
@@ -25,4 +25,7 @@ compiler-rt = { path = "compiler-rt" }
 c = []
 weak = ["rlibc/weak"]
 
+[profile.dev]
+debug-assertions = false
```

Comment:
It seems desirable to leave this on, right? It's caught bugs in overflows and such?